Gemfile

```diff
@@ -74,6 +75,38 @@ else
       gem 'msgpack', '1.7.2'
     end
   end
+else
+
+  puts "FINAL_PACKAGE"
+
+  gem 'oslg'
+
+  if !MINIMAL_GEMS
+    gem 'tbd'
+    gem 'osut'
+
+    gem 'openstudio-extension', '= 0.7.1'
+    gem 'openstudio-workflow', '= 2.3.1'
+    gem 'openstudio_measure_tester', '= 0.3.3'
```

Comment:
And so here I do NOT specify :git or :path on purpose, so it jsut grabs the "vendor/cache/openstudio....gem" I built and asks no question, since I install --local